src/main/java/Victor/TaskType/Deadline.java

```diff
@@ -0,0 +1,29 @@
+package Victor.TaskType;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected String by;
+    protected LocalDate tempBy;
```

Comment:
It may just be me, but I didn't understand the purpose of this variable from the name.

Reply:
The tempBy is used as temporary storage of the date, used to hold the date in a LocalDate format, while the by holds the date in a String format. The by is stored in the data file, while the tempBy is normally used to just print out the date in LocalDate format.